src/notation/view/pianokeyboard/pianokeyboardcontroller.h

```diff
@@ -32,6 +32,7 @@
 namespace mu::notation {
 class PianoKeyboardController : public muse::Injectable, public muse::async::Asyncable
 {
+    muse::Inject<engraving::IEngravingConfiguration> configuration = { this };
```

Comment:
(when injecting a Configuration from a different module, we would usually make that clear in the name, like `engravingConfiguration`)